consensus/core/src/block.rs

```diff
@@ -112,6 +139,24 @@ impl BlockRef {
             digest,
         }
     }
+
+    pub fn round(&self) -> Round {
```

Comment:
round, author and digest are already pub fields. Is it necessary to add the accessor methods?